client/src/components/common/boost/BoostConfigurator.tsx

```diff
@@ -191,6 +201,7 @@ export default function BoostConfigurator({
             >
               Upload All Configs
             </Button>
+            <BoostResult results={boostResult} />
```

Comment:
How would you feel about hiding this completely if `boostResult` is `null`? Alternatively, we could provide a short message saying there is no current plan generated.